ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -2671,6 +2689,12 @@ literals:
 - refrigerant_suction_superheat_temperature_sensor:
     flexible_min: 255.9259259
     flexible_max: 422.037037
+- refrigerant_injection_suction_superheat_temperature_sensor:
```

Comment:
what makes these points injection sensors? is this sensor seperate from the one above?